composer.json

```
@@ -4,13 +4,13 @@
     "type": "sulu-bundle",
     "description": "Bundle that provides controllers and services for using Sulu as headless content management system",
     "require": {
-        "php": "^7.3 || ^8.0",
-        "sulu/sulu": "^2.4 || ^2.5@dev",
-        "symfony/config": "^4.4 || ^5.0 || ^6.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^4.4 || ^5.0 || ^6.0",
-        "symfony/http-foundation": "^4.4 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^4.4 || ^5.0 || ^6.0"
+        "php": "^8.2 || ^8.3",
+        "sulu/sulu": "^2.4 || ^2.5@dev || ^2.6@dev",
```

```suggestion
        "sulu/sulu": "^2.4 || ^2.5|| ^2.6@dev",
```

The `@dev` we use on all our bundles to test the CI always against `2.6` branch instead of the released versions on highest independently what minimum-stability is configured.

---

oh yeah :D